scripts/dev-bundle-tool-package.js

```diff
@@ -12,43 +12,44 @@ var packageJson = {
     // and we want to make sure there are no dependencies on a higher version
     npm: "6.14.8",
     pacote: "https://github.com/meteor/pacote/tarball/a81b0324686e85d22c7688c47629d4009000e8b8",
-    "node-gyp": "6.0.1",
-    "node-pre-gyp": "0.14.0",
-    typescript: "3.8.3",
+    "node-gyp": "6.1.0",
+    "node-pre-gyp": "0.15.0",
+    typescript: "3.9.5",
```

Comment:
How about going straight for 4.0? It was already requested in https://github.com/meteor/babel/pull/33 and https://github.com/meteor/meteor/pull/11136. Breaking changes are also [fairly _breaking_](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#breaking-changes).

Reply:
I don't know enough about TypeScript or its implementation here to risk a major version upgrade. Probably best if someone with that knowledge makes a separate PR which will cover both the internal version and the one in packges.

Reply:
I believe typescript update to 4 will be ready first so maybe it's better to not include here directly.